common/federation/src/test/java/app/coronawarn/server/common/federation/client/CloudFederationFeignHttpClientProviderSmokeTest.java

```diff
@@ -29,4 +31,32 @@ void testCanLoadKeystore() {
     assertThat(cut.createFeignClient()).isNotNull();
   }
 
+  @Test
+  void testCertificate(){
+    Ssl ssl = new Ssl();
+    ssl.setCertificateDn("X.509");
+    assertThat(ssl.getCertificateDn(),is("X.509"));
+  }
+
+  @Test
+  void testCertificateSHA(){
+    Ssl ssl = new Ssl();
+    ssl.setCertificateSha("randomString123123");
+    assertThat(ssl.getCertificateSha(),is("randomString123123"));
+  }
+
+  @Test
+  void testGetBaseUrl(){
+    Ssl ssl = new Ssl();
+    ssl.setBaseUrl("https://github.com/cwa-server/tree/main");
+    assertThat(ssl.getBaseUrl().isNotNull());
+  }
+
+  @Test
+  void testGetCertificationType(){
+    Ssl ssl = new Ssl();
+    ssl.setCertificateType("PKCS12");
+    assertThat(ssl.getCertificateType(),is("PKCS12"));
```

Comment:
Instead of using `assertThat(x, is("y"))`, this projects tends to use `assertThat(x).isEqualTo("y")`